src/App.ts

```diff
@@ -565,19 +565,19 @@ export default class App {
         type === IncomingEventType.Event
           ? (bodyArg as SlackEventMiddlewareArgs['body']).event
           : type === IncomingEventType.ViewAction
-          ? (bodyArg as SlackViewMiddlewareArgs['body']).view
```

Comment:
I think Prettier did a lot of these formatting changes. Personally, I'm not a fan of most of them. But I guess the point in using Prettier was to take personal opinions out of it, right?